tests/smoke.runner.js

```diff
@@ -35,7 +35,9 @@ async function runTests (tests) {
         }
     } else {
         // parallel
-        await Promise.all(tests.map(test => test()))
+        await Promise.all(tests.map(test => test().catch(() => {
```

Comment:
will it catch just one failed test? Maybe, here might be better to use `Promise.allSettled()`?

Reply:
I would prefer to stop running other tests once one smoke test is failing.

Reply:
we cant use allSettled because it was added in nodejs 12 !!